students/shibin_mathew/lesson9/mailroom.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+import sys
+from collections import defaultdict
+
+
+class Donor:
+    def __init__(self, names, amount):
+        self.names = names
+        self.amount = amount
+
+    @property
+    def names(self):
+        return "{}".format(self._names)
+
+    @names.setter
+    def names(self, names):
+        self._names = names
+
+    @property
+    def amount(self):
+        return self._amount
+
+    @amount.setter
+    def amount(self, amount):
+        self._amount = amount
```

Comment:
if you are not doing anything special you don't really need to use properties, just regular class attributes should work.

Reply:
Good point - I made the change.